doc/specs/OOBE-Dialog.md

```diff
@@ -0,0 +1,169 @@
+
+### 1.2. Key Definitions/Concepts
+
+Here are definitions of words and acronyms found throughout this document to ensure clarity:
+
+- **OOBE:** Out of box experience – The users’ initial interactions with the product immediately after installing the product and/or launching the product for the first time.
+
+### 1.3. Goals and Non-Goals
+
+Goals:
+
+- Create a guided prompt that exposes the user to a brief overview of each PowerToy, highlighting the PowerToys’ basic usage.
+- Present a short-list of useful customization options that the user can select to define their experience.
+
+Non-goals:
+
+- Present an exhaustive list of features and options that are available in PowerToys. This OOBE prompt is only meant to provide initial exposure for which interested users should continue investigating via provided links to documentation and settings.
+
+## 2. Definition of Success
+
+### 2.1. Customers
+
+The PowerToys OOBE is for new power users and developers who are looking to tune and streamline their Windows experience for greater productivity and enhanced user experience. As the PowerToys customer base tends to be particularly biased against OOBE prompts in general, we need to present the PowerToys OOBE dialog in such a way that it provides immediate value to end-users to improve the likelihood of users discovering all the PowerToys by completing the prompts.
+
+### 2.2. Expected Impact: Customer, and Technology Outcomes
+
+- **High Reliability:** Less than 0.1% crash rate.
+- **Increased Activation:** 30% or more increase in usage in all PowerToys features.
+- **High User Retention:** 25% or more active PowerToys users after 28 days.
+- **High OOBE Completion Rate:** 50%  or more of New PowerToys users reach the last page of the OOBE prompt.
+
+## 3. Requirements
+
+The base of this OOBE dialog was attributed by [Niel’s mock-up](https://github.com/microsoft/PowerToys/issues/1285).
+
+### 3.1. Functional Requirements
+
+#### 3.1.1. Functional Requirements Overview
+
+|No. | Requirement | Pri |
+| - | - | - |
+|1 | The OOBE Dialog window should launch immediately after installation. | P0 |
+|2 | The OOBE Dialog should consist of a collection of pages that describe the available PowerToys. | P0 |
+|3 | On launch, present the Welcome/Landing page in the OOBE Dialog window based on the template described in [Figure 5.1.1.](#511-oobe-dialog-layout) | P1 |
+|4 | Traverse the various OOBE Dialog pages via an list view options on the left-hand pane. | P0 |
+|5 | Alternatively traverse the various OOBE Dialog pages via an on-screen next/previous buttons on each page. | P1 |
+|6 | At any time, the user should be able to exit the OOBE dialog by closing the window. | P0 |
+|7 | Alternatively, the user should be able to exit the OOBE dialog by pressing the ESC key. | P1 |
+|8 | When the OOBE dialog is exited, save all the options that were set by the user. | P0 |
```

Comment:
It's not clear what is the expected behavior, since any change made in the Settings pages is instantly saved, so what exactly is the expected behavior here? That we disable instant-save in Settings and we somehow cache the changes to then apply them only when the OOBE dialog is exited? I hope not because it would require too much work for basically no benefit.

Reply:
That's a good point of clarification, the settings should be saved automatically upon being changed. I'll update the verbiage to reflect that.